src/index.ts

```diff
@@ -78,6 +86,66 @@ export function createSignature(
     return new Signature(KeyType.WA, encoder.getBytes())
 }
 
+export function recoverPublic(signature: Signature, message: Bytes): PublicKey {
+    const signatureData = signature.data.array
+    const messageDigest = Checksum256.hash(message).array
+
+    const recid = signatureData[0] - 31
```

Comment:
Should we add a quick check to make sure that the recId is in range (basically that it's above 0)?

Reply:
I think i don't know this value's llegal range. Do you know what materials we can refer to here?